molecule/default/playbook.yml

```diff
@@ -3,6 +3,8 @@
   become: true
   vars:
     pulp_config_secret_key: secret
+    pulp_default_admin_password: ''
```

Comment:
Doesn't this break idempotence? It seems wonky for a single PR to fix an idempotence issue arising from `pulp_default_admin_password` being set, and to also set `pulp_default_admin_password`.

Reply:
actually it fixes it :smile:
The problem is if you set `pulp_default_admin_password` then the role will try update admin password (roles/pulp3/tasks/database.yml, line 92) with `pulp-manager reset-admin-password`  but that command is not idempotence and currently I don't have an idea for elegant solution.